app/_hub/kong-inc/oauth2/how-to/_oauth2rla.md

```diff
@@ -0,0 +1,479 @@
+       "global_credentials": false,
+       "refresh_token_ttl": 1209600,
+       "token_expiration": 7200
+     },
+     "ordering": null
+   }
+   ```
+1. Create a consumer named `user1`.
+   ```bash
+   curl --request POST \
+  --url http://localhost:8001/teamA/consumers \
+  --header 'Content-Type: multipart/form-data' \
+  --header 'kong-admin-token: admin' \
+  --form username=user1
+   ```
+   The results should look like this:
+  ```txt
+   {
+     "username": "user1",
+     "updated_at": 1699526969,
+     "tags": null,
+     "created_at": 1699526969,
+     "custom_id": null,
+     "username_lower": "user1",
+     "type": 0,
+     "id": "691d8afc-b098-4374-937b-b1c36c8ef084"
+   }  
+  ```
+1. Create a key with the value `123` for the consumer `user1`.
+   ```bash
+   curl -i -X POST localhost:8001/teamA/consumers/user1/key-auth \
+--data "key=123"
+   ```
+   The results should look like this:
+   ```txt
+   {
+      "consumer":{
+         "id":"691d8afc-b098-4374-937b-b1c36c8ef084"
+      },
+      "ttl":null,
+      "created_at":1699527247,
+      "key":"123",
+      "id":"233005b4-3d03-4618-9de9-e05887dd2055",
+      "tags":null
+   }
+   ```
+1. Provision new OAuth 2.0 credentials with `client_id` set to `456` and `client_secret` set to `789`.
+   ```bash
+   curl -i -X POST http://localhost:8001/teamA/consumers/user1/oauth2 \
+--data "name=demo" \
+--data "client_id=456" \
+--data "client_secret=789" \
+--data "redirect_uris=http://mockbin.org/"
+   ```
+   The results should look like this:
+   ```txt
+   {
+      "tags":null,
+      "created_at":1699527585,
+      "redirect_uris":[
+         "http://mockbin.org/"
+      ],
+      "consumer":{
+         "id":"691d8afc-b098-4374-937b-b1c36c8ef084"
+      },
+      "hash_secret":false,
+      "name":"demo",
+      "client_type":"confidential",
+      "id":"77dec565-8b1b-4e60-8ad1-59fa1a6f280e",
+      "client_id":"456",
+      "client_secret":"789"
+   }   
+   ```
+1. Get a token for the client credentials.
+   ```bash
+   curl --request POST -k \
+  --url 'https://localhost:8443/mockbin/oauth2/token' \
+  --header 'Content-Type: multipart/form-data' \
+  --header 'kong-admin-token: admin' \
+  --form client_id=456 \
+  --form client_secret=789 \
+  --form grant_type=client_credentials | jq
+   ```
+   The results should look like this:
+   ```txt
+   {
+  "token_type": "bearer",
+  "expires_in": 7200,
+  "access_token": "k55RovztGnAHwl8xC07gfbuC345TEDRY"
+   }
+   ```
+   Make a note of this `access_token` to send proxy requests to the service.
+
+1. Create a service named `Oauth` with `oauth2/token` endpoint.
+   ```bash
+  curl --request POST \
+    --url http://localhost:8001/teamA/services \
+    --header 'Content-Type: application/json' \
+    --header 'Kong-Admin-Token: admin' \
+    --data '{
+  	"name": "Oauth",
+  	"retries": 0,
+  	"host": "localhost",
+  	"path": "/mockbin/oauth2/token",
+  	"port": 8443,
+    "protocol":"https"
+  }' | jq
```

Comment:
```suggestion
    --url http://localhost:8001/teamA/services \
    --data "name=Oauth" \
    --data "retries=0" \
    --data "host=localhost" \
    --data "path=/mockbin/oauth2/token" \
    --data "port=8443" \
    --data "protocol=https"
```
Why is this one in application/json format? Please convert to `--data` flags as in previous examples on this page. Make sure to test that this will still work.